### DISCUSSION TITLE
Req: EU AQI and Pollen Counts

### DISCUSSION THREAD

**Discussion by AdamHayter** [rocksdanister/weather#74] [Ideas]
Hello,
I checked out the Open-Meteo API and there is the ability to include Pollen as part of the Air Quality API in supported locations (I'm europe based so the data is available)

It would be very nice to see the 'Air Quality' box have two scales; One for general air quality, and (if available) one for pollen counts.
Also an option to use EU AQI instead of US AQI would be great!

**Comment by rocksdanister** [comment]
moved here: https://github.com/rocksdanister/weather/issues/75

I will add the pollen count to the flyout when the card is clicked.
EU AQI is complicated since it will require calculation for various providers.
